WordPress/Classes/ViewRelated/EEUUSCompliance/CompliancePopoverViewController.swift

```diff
@@ -0,0 +1,83 @@
+import UIKit
+import SwiftUI
+import WordPressUI
+
+final class CompliancePopoverViewController: UIViewController {
+
+    // MARK: - Views
+    private lazy var hostingController: UIHostingController = {
+        let controller = UIHostingController(rootView: CompliancePopover(viewModel: self.viewModel))
+        controller.view.translatesAutoresizingMaskIntoConstraints = true
+        return controller
+    }()
```

Comment:
Does this need to be `lazy`? This property is guaranteed to be used by `CompliancePopoverViewController`, so it should probably be declared as a `let` and set up in the `init` method.

Overusing `lazy` can make initialization order difficult to follow, which is more likely to cause bugs.